src/gei/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -55,13 +55,19 @@ public async Task Handle(MigrateRepoCommandArgs args)
 
         if (args.GhesApiUrl.HasValue())
         {
-            var targetExists = await _targetGithubApi.DoesRepoExist(args.GithubTargetOrg, args.TargetRepo);
+            var targetRepoExists = await _targetGithubApi.DoesRepoExist(args.GithubTargetOrg, args.TargetRepo);
+            var targetOrgExists = await _targetGithubApi.DoesOrgExist(args.GithubTargetOrg);
 
-            if (targetExists)
+            if (targetRepoExists)
             {
                 throw new OctoshiftCliException($"A repository called {args.GithubTargetOrg}/{args.TargetRepo} already exists");
             }
 
+            if (!targetOrgExists)
+            {
+                throw new OctoshiftCliException($"The target org, {args.GithubTargetOrg}, does not exist. Migration unable to continue at this time.");
```

Comment:
IMO the `migration is unable to continue...` is a bit redundant since we're already throwing an exception and telling the user that we cannot continue 😃  

```suggestion
                throw new OctoshiftCliException($"The target org \"{args.GithubTargetOrg}\" does not exist.");
```

Reply:
Fair point! Fixed here: c091ce9ac66d3dbfbdfa36db43f670651c5fc37a